alfresco-solrapi-client/alfresco-solrapi-client-api/src/main/java/eu/xenit/alfresco/client/solrapi/api/model/SolrNodeMetaData.java

```diff
@@ -1,37 +1,34 @@
 package eu.xenit.alfresco.client.solrapi.api.model;
 
-import java.util.ArrayList;
-import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
-import lombok.Data;
+import lombok.Value;
 import lombok.experimental.Accessors;
 
-@Data
+@Value
 @Accessors(chain = true)
 public class SolrNodeMetaData
 {
-    private long id = -1L;
-    private long aclId = -1L;
-    private long txnId = -1L;
```

Comment:
this causes it to default to 0L
This will fail an integration test

If we want the default to be 0L, then we shoud also set it like that in Ditto, else we can't reuse the integration test across both implementations

Reply:
I intended to make all fields `final` with the `@Value` annotation.

> `@Value` is the immutable variant of `@Data`; all fields are made private and final by default

I think we need to fix that where the constructor is called ?